docs/src/examples.md

```diff
@@ -25,7 +25,7 @@ Follows the continuity equation
 This is discretized using the following
 ```math
 \begin{equation}
-  \frac{\partial}{\partial t} \rho \approx - D_h[ \rho (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] - D^c_v[I^f(\rho \boldsymbol{u}_h)) + I^f(\rho) \boldsymbol{u}_v)] .
+  \frac{\partial}{\partial t} \rho \approx - wD_h[ \rho (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] - D^c_v[I^f(\rho \boldsymbol{u}_h)) + I^f(\rho) \boldsymbol{u}_v)] .
```

Comment:
is this change still relevant?

Reply:
Yes, this is reflecting the current the 3D Box case code and I am going to update it in a follow up PR, where I will change it to use strong div